packages/story-editor/src/elements/video/display.js

```diff
@@ -118,19 +119,7 @@ function VideoDisplay({ previewMode, box: { width, height }, element }) {
           data-leaf-element="true"
         >
           {url && <source src={url} type={resource.mimeType} />}
-          {tracks &&
-            tracks.map(({ srclang, label, track: src, id: key }, i) => (
-              <track
-                srcLang={srclang}
-                label={label}
-                // Hides the track from the user.
-                // Displaying happens in MediaCaptionsLayer instead.
```

Comment:
This useful comment is now lost

Reply:
Is this really a problem?

Reply:
It helps explain why it‘s needed, so yes